canonical-websites/snapcraft.io
# Create snapcraft blueprint

# Summary

This PR is based on my previous PR: https://github.com/canonical-websites/snapcraft.io/pull/718
Just look at the last commit until the other PR is not merged

- Move snapcraft specific pages to blueprint `snapcraft`

# QA

- `./run`
- http://127.0.0.1:8004/
- http://127.0.0.1:8004/create
- http://127.0.0.1:8004/docs
- http://127.0.0.1:8004/favicon.ico
- http://127.0.0.1:8004/community
- Make sure those links still work
